sphinx/ext/autosummary/generate.py

```diff
@@ -630,6 +645,10 @@ def get_parser() -> argparse.ArgumentParser:
                         dest='imported_members', default=False,
                         help=__('document imported members (default: '
                                 '%(default)s)'))
+    parser.add_argument('-a', '--respect-module-all', action='store_false',
+                        dest='ignore___all__', default=True,
```

Comment:
`ignore_all` ?

Reply:
Renamed to `respect_module_all` and flipped the boolean, then added a not to line 668